apps/protocol-frontend/src/components/MyAttestationsTable.tsx

```diff
@@ -22,41 +21,34 @@ import {
   useTable,
   UseTableHooks,
 } from 'react-table';
-import { useUser } from '../contexts/UserContext';
 import GlobalFilter from './GlobalFilter';
 import { formatDate } from '../utils/date';
+import InfiniteScroll from 'react-infinite-scroll-component';
 import { UIContribution } from '@govrn/ui-types';
+import { GovrnSpinner } from '@govrn/protocol-ui';
 
 type MyAttestationsTableType = {
   id: number;
   date_of_submission: Date | string;
   date_of_engagement: Date | string;
   status: string;
   name: string;
-  attestationDate: string;
+  attestationDate: Date | string;
   contributor?: string | null;
 };
 
 const MyAttestationsTable = ({
   contributionsData,
-}: // setSelectedContributions,
-{
+  hasMoreItems,
+  nextPage,
+}: {
   contributionsData: UIContribution[];
+  hasMoreItems: boolean;
+  nextPage: () => void;
 }) => {
-  const { userData } = useUser();
-
-  const nonEmptyContributions = _.filter(
-    contributionsData,
-    a => a.attestations?.length > 0,
-  );
-
-  const attestedContributions = _.filter(nonEmptyContributions, a =>
-    a.attestations.every(b => b.user_id === userData?.id),
-  );
-
   const data = useMemo<MyAttestationsTableType[]>(
     () =>
-      attestedContributions.map(contribution => ({
+      contributionsData.map(contribution => ({
```

Comment:
May not be something we worry about right now, but we may want to add a similar 'Empty Contributions' component that we use in the Attestations Table for when this is 0. Something for us to consider

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/9438776/195364929-d56df218-ec85-436d-80b1-be31284fe9c4.png">

Reply:
This will be fixed as well